lib/supplejack_common/paginated_collection.rb

```diff
@@ -103,7 +104,7 @@ def more_results?
       if tokenised?
         return klass.next_page_token(@next_page_token_location).present?
       end
-      current_page <= total_pages
+      current_page < total_pages
```

Comment:
should we add a spec to cover this?

we can probably have a spec that expects the method `fetch_records` to be called as many times based on the `current_page` and `total_pages`

around here:
https://github.com/DigitalNZ/supplejack_common/blob/916d9a456ae0347cd4c7bac59286c16719849784/spec/supplejack_common/paginated_collection_spec.rb#L27